packages/lesswrong/lib/vulcan-lib/components.tsx

```diff
@@ -109,6 +113,10 @@ const addClassnames = (componentName: string, styles: any) => {
   })
 }
 
+export const useStyles = (styles: (theme: ThemeType)=>JssStyles, componentName: keyof ComponentTypes) => {
```

Comment:
Not sure what this change is doing and why it's only in lw-deploy

Reply:
Fine to bring in, it will be unused but still functional and there's reason to think we'll use it later in other contexts.